sdk/core/core-http/README.md

```diff
@@ -23,15 +27,51 @@ Examples can be found in the `samples` folder.
 
 ## Next steps
 
+First obtain an access token to the management service used in the sample code
```

Comment:
At this point, the user is not aware of which management service is used in the sample code. Are there multiple management services and we are using one in the sample? Is there is single service? What service is this?

Reply:
This is to test that the created service client works. I believe the old code was trying to use an azure management service end point to list some resources.  It doesn't really matter what service is used here. Also since core-http is mainly used in code gen, to me this sample is more for us to use than for users.

The old code uses access token. I was thinking that DefaultAzureCredential might work here too. But that would also require some setup.